### ISSUE TITLE
yuv4mpegpipe: ERROR: Codec not supported

### ISSUE THREAD

**Issue by Qub3k** [VQEG/siti-tools#20]
I am working with 3840x2176 YUV videos with 10-bit data in the yuv420p10le pixel format. Following the recommendation in the README file, I decided to convert the videos to the Y4M format. Thus, I ran a modified version of the FFmpeg call refered to in the README:

```
ffmpeg.exe -hide_banner -f rawvideo -pix_fmt yuv420p10le -framerate 50 -video_size 3840x2176 -i input.yuv -c:v copy output.y4m
```

However, after I did that, FFmpeg issued the following error message:
```
[yuv4mpegpipe @ 0000012a90b07ac0] ERROR: Codec not supported.
Could not write header for output file #0 (incorrect codec parameters ?): Invalid data found when processing input
Error initializing output stream 0:0 --
```

Thus, I had to drop the `-c:v copy` parameter and use the following command to put my raw content in the Y4M container:

```
ffmpeg.exe -hide_banner -f rawvideo -pix_fmt yuv420p10le -framerate 50 -video_size 3840x2176 -i input.yuv -f yuv4mpegpipe -strict -1 output.y4m
```

(The `-strict -1` parameter was necessary due to the yuv420p10le pixel format of the input.)

Although the FFmpeg will report the output video stream format as _wrapped_avframe_, the output stream contains an umodified copy of the input raw video.

**Thus, my suggestion is to mention in the README file that skipping the `-c:v copy` argument may be necessary to put your raw video in the Y4M container. I am happy to create a related pull request with the necessary changes in the README file if you think this is of value.**

(I am using the following version of FFmeg: `5.1.2-full_build-www.gyan.dev`.)

**Comment by slhck** [comment]
Yes, please provide a PR if you can! Thank you!

**Comment by slhck** [comment]
Fixed in 9dbf775
